toolkit/resources/manifests/package/update_manifests.sh

```diff
@@ -306,6 +306,7 @@ generate_pkggen_core () {
         grep "^msopenjdk-" $TmpPkgGen
         grep "^pyproject-" $TmpPkgGen
         grep "^audit-" $TmpPkgGen
+        grep "^libcap-ng-" $TmpPkgGen
```

Comment:
This is closer to a nit, but we should put this line before the one with `util-linux`, given that `libcap-ng` is a prerequisite for `util-linux`. The order of the pkggen_core manifest is *supposed* to be the order RPMs are installed in the chroot. I'm not sure how true that is these days, and it's not strictly necessary due to the way we install RPMs in the chroot, but I think we should still strive to keep this ordered as well as we can.